app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    command = command.split()
+    print(command)
+    if command[1] == command[2]:
+        pass
+    elif ".txt" in (command[1] and command[2]):
+        with (open(command[1], "r") as file_in,
+              open(command[2], "w") as file_out):
+            file_out.write(file_in.read())
```

Comment:
Code Style: The variable names 'file_in' and 'file_out' could be more descriptive. Consider using 'source_file' and 'destination_file' instead. This would make your code more readable.